parse.jai

```diff
@@ -2789,18 +2789,17 @@ parse_perl_flags :: (using ps: *ParseState, s: *string) -> Status, success: bool
 	if (t.count > 2 && t[0] == #char "P" && t[1] == #char "<") {
 		advance(*t, 2);
 		// Pull out name.
-		end, found := index_of_char(t, #char ">");
-		if !found {
+		end := find_index_from_left(t, #char ">");
+		if end == -1 {
 			valid: bool;
 			valid, status = is_valid_utf8(<<s);
 			if !valid		return status, false;
-
 			return make_status(.BadNamedCapture, <<s), false;
 		}
 
 		// t is "P<name>...", t[end] == '>'
 		capture := slice(<<s, 0, end + 3);
-		name := slice(t, 2, end - 2);
+		name := slice(t, 0, end);
```

Comment:
This change seems wrong. (But I haven’t looked too hard at it.)
What’s the reason for it?

Reply:
Nope, you were right. The comment above it was wrong (due to the `advance(t, 2)` a bit further up).